DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: redshiftTools
 Type: Package
 Title: Redshift Tools
-Version: 0.1.2
+Version: 0.2.2
```

Comment:
Just curious: Do we have some kind of a version numbering scheme we're following? The change makes me wonder that there's probably a reason why we're going from 0.1.2 to 0.2.2 and not e.g. 0.2.0. And I should maybe be aware of the reason.

Reply:
@ilarischeinin That was a mistake on my part.  It should be 0.2.0.  In general I've been leaning away from full-on [semantic versioning](https://semver.org/) towards date versioning with 'incompatible API changes' resulting in a new package.  But 🤷 I'm flexible.

Reply:
Come to think of it - there could be broken APIs here, it probably should have been a Major bump. 😬  But, for the moment, I just want to get it through to shipped - and there aren't a lot of remaining stuff that depends on the changed items that I'm worried about it.